.github/workflows/docker.yml

```diff
@@ -41,6 +41,24 @@ jobs:
           else
             echo git_branch="$GITHUB_REF_NAME" >> $GITHUB_OUTPUT
           fi
+      - name: Free Disk Space (Ubuntu)
```

Comment:
We could also maybe provide a flag to make this step optional since only 1 repo is known to be failing?